### PR TITLE
[aws-sdk-cpp] Remove `compute_build_only.cmake`.

### PR DESCRIPTION
**Pull request by teo-tsirpanis**
The only thing it does is to convert the targets to lowercase, which can be done easily from the portfile.

- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [ ] SHA512s are updated for each updated download
- [ ] The "supports" clause reflects platforms that may be fixed by this new version
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ ] Any patches that are no longer applied are deleted from the port's directory.
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/aws-sdk-cpp/portfile.cmake [hunk 1]
```diff
@@ -29,8 +29,8 @@ else()
     set(rpath "\$ORIGIN")
 endif()
 
-set(BUILD_ONLY core)
-include(${CMAKE_CURRENT_LIST_DIR}/compute_build_only.cmake)
+set(BUILD_ONLY core ${FEATURES})
+string(TOLOWER ${BUILD_ONLY} BUILD_ONLY)
```
Comment (dg0yt) [inline ports/aws-sdk-cpp/portfile.cmake:33]: Isn't this just
```suggestion
set(BUILD_ONLY "${FEATURES}")
```

**Comment by Cheney-W** [comment]
```
CMake Error at D:/installed/x64-windows/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake:36 (message):
  'D:/packages/aws-sdk-cpp_x86-windows/debug/lib/cmake/aws-cpp-sdk-core' does
  not exist.
```
@teo-tsirpanis Could you please solve this error?

File: ports/aws-sdk-cpp/portfile.cmake (hunk 1 above)
Comment (teo-tsirpanis) [reply ports/aws-sdk-cpp/portfile.cmake]: Thought `core` was necessary, turns out it's not. And I replaced `BUILD_ONLY` with `FEATURES`.

Comment (dg0yt) [reply ports/aws-sdk-cpp/portfile.cmake]: FTR `core` is always included in `FEATURES`.

**Comment by teo-tsirpanis** [comment]
Should be ready to merge.

**Review by Cheney-W — DISMISSED** [review]

**Review by BillyONeal — DISMISSED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**